deploy/eck-elasticsearch/values.yaml

```diff
@@ -28,6 +28,10 @@ labels: {}
 #
 annotations: {}
 
+# Control the Elasticsearch Users and Roles
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-users-and-roles.html
```

Comment:
```suggestion
# Settings for configuring Elasticsearch users and roles.
# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-users-and-roles.html
#
```

Reply:
Thank you for the review. 

unit test has been added, also updated the description.